x-pack/plugins/observability_solution/inventory/public/plugin.tsx

```diff
@@ -49,6 +51,28 @@ export class InventoryPlugin
   ): InventoryPublicSetup {
     const inventoryAPIClient = createCallInventoryAPI(coreSetup);
 
+    pluginsSetup.observabilityShared.navigation.registerSections(
+      from(coreSetup.getStartServices()).pipe(
+        map(([coreStart, pluginsStart]) => {
+          return [
+            {
+              label: '',
+              sortKey: 101,
+              entries: [
+                {
+                  label: i18n.translate('xpack.inventory.inventoryLinkTitle', {
+                    defaultMessage: 'Inventory',
+                  }),
+                  app: INVENTORY_APP_ID,
+                  path: '',
```

Comment:
Do we need a path here?

Reply:
it also works without but I'll specify `"/" `

Reply:
https://github.com/elastic/kibana/pull/193251/commits/ff54d98ca4dd7f8a12040b622bb8cb82ffe1dcd7#diff-09476b1a2b68b7d93ef695f5314086f1483505b44d10ba5df2902c6b58cc693eR66